src/screens/SignedChatScreen/index.tsx

```diff
@@ -73,12 +77,6 @@ const SignedChatScreen = () => {
     }
   }, [selectedChannel]);
 
-  React.useEffect(() => {
-    return () => {
-      clearReplyPreview();
-    };
-  }, []);
-
   return (
     <ScrollContext.Provider value={scrollContext}>
       <View style={styles.keyboardAvoidingView}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `clearReplyPreview` call has been removed from the `useEffect` hook. If there's no other place where `clearReplyPreview` is being called when the component unmounts, this could lead to unwanted state persistence across component instances.
```diff
- React.useEffect(() => {
-   return () => {
-     clearReplyPreview();
-   };
- }, []);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->